fe/fe-core/src/main/java/org/apache/doris/datasource/hive/event/MetastoreEventsProcessor.java

```diff
@@ -125,6 +128,8 @@ private void realRun() {
                 } catch (MetastoreNotificationFetchException e) {
                     LOG.warn("Failed to fetch hms events on {}. msg: ", hmsExternalCatalog.getName(), e);
                 } catch (Exception ex) {
+                    hmsExternalCatalog.onRefresh(true);
```

Comment:
What is this 2 lines for?

Reply:
Suppose you receive a set of events, and when processing the insert event, the initialization reports a nullpointer. The event processing fails, and our data with Hive is out of sync? Refresh it and you can process other events next time. i will modify it to `onRefreshCache`